jekyll/_cci2/credits.md

```diff
@@ -35,15 +35,70 @@ To set up your desired plan, go to `Settings > Plan Overview` in the CircleCI we
 ## Free plan
 {: #free-plan }
 
-As with the CircleCI legacy Container plan, CircleCI also supports a free-tier with the usage-based plan. You can still make use of many key features offered by CircleCI:
-
-- Using Orbs
-- Workspaces
-- Dependency Caching
-- Windows/Linux builds
+As with the CircleCI legacy Container plan, CircleCI also supports a free-tier with the usage-based plan. By using the Free Plan, you can take advantage of a large number of premium features that will allow your team to be more productive, efficient and fast.
 
 The free usage-based plan offers a set amount of build credits across medium-type machines per week (which offers 2 CPUs, 4gb of memory). With this combination, a small number of credits are charged per minute and there is no limit on user seats. Refer to the [Pricing](https://circleci.com/pricing/) page for more information on credit amounts.
 
+The table below lists some of the features you will be able to use on the Free Plan.
+
+| Feature | Description |
+|-------|-------------|
+| UNLIMITED USERS | You may have an unlimited number of users. This enables you to have different teams and groups working together on projects |
+| EXPANSIVE LIST OF AVAILABLE RESOURCE CLASSES | Docker, Linux and Windows are available (macOS coming soon) are available for us |
+| CONCURRENCY AND TEST SPLITTING | Faster builds and test splitting equals increased productivity and speed when building on CircleCI |
+| SELF HOSTED RUNNERS | Increased number of runners you can use when working on CircleCI |
+| FLAKY TEST DETECTION | Improved inisghts are available to help analyze and detect flaky tests |
+| UNLIMITED PRIVATE ORBS | Streamline team collaboration by sharing CircleCI Orbs across teams and groups |
+| DOCKER LAYER CACHING | Efficiency features designed specifically for Docker, providing improved speed and efficiency |
+{: class="table table-striped"}
+
+### Unlimited users
+{: #unlimited-users }
+
+When you use the free plan, you may have an unlimited number of users building and developing on CircleCI. This allows you to collaborate with more teams and groups within your organization, which can increase speed, productivity, and efficiency.
+
+### Expansive list of available resource classes
+{: #expansive-list-of-available-resource-classes }
+
+When using the free plan, you have the widest array of resource classes on Docker, Linux, Windows and macOS (coming soon) available to use. With this flexibility in choosing resource classes, you can be sure to choose the right compute resources for your job.
```

Comment:
```suggestion
When using the Free plan, you have the widest array of resource classes on Docker, Linux, Windows and macOS (coming soon) available to use. With this flexibility in choosing resource classes, you can be sure to choose the right compute resources for your job.
```
I think I'd also shorten the second sentence to just something like, "This flexibility helps ensure that you are choosing the right compute resources..." 

Also are we referring to `job` as in the CircleCI concept of job?